### PR TITLE
control-service: Document variables supported for configuring logs UR…

### PR DESCRIPTION
**Pull request by mivanov1988**
Added helm configuration for logs URL format/template and document supported variables.

Testing Done: manual deployment

Signed-off-by: Miroslav Ivanov miroslavi@vmware.com

### CODE REVIEW

File: projects/control-service/projects/helm_charts/pipelines-control-service/values.yaml [hunk 1]
```diff
@@ -810,3 +810,37 @@ dataJobExecutionsCleanupTask:
   maximumExecutionsToStore: "100"
```
Comment (tpalashki) [inline projects/control-service/projects/helm_charts/pipelines-control-service/values.yaml:810]: Is there a reason to keep the old settings?

File: projects/control-service/projects/helm_charts/pipelines-control-service/values.yaml [hunk 2]
```diff
@@ -810,3 +810,37 @@ dataJobExecutionsCleanupTask:
...
+    ## The execution API helps you collect information about previous
+    ## data job executions, or manually start a job execution. Stored data job
+    ## execution information will naturally increase over time. This is why we
+    ## are providing a way to clean old information about data job executions through
+    ## the following properties:
+    ##
+    ## maximumExecutionsToStore - The maximum amount of executions per data job to keep. If data
+    ##                            job executions exceed this number, the older executions are deleted.
+    ##
+    ## ttlSeconds - Total time to live per data job execution. Data job executions older than ttlSeconds
```
Comment (tpalashki) [inline projects/control-service/projects/helm_charts/pipelines-control-service/values.yaml:826]: Rename this to match the property.

**Review by tpalashki — APPROVED** [review]

File: projects/control-service/projects/helm_charts/pipelines-control-service/values.yaml (hunk 1 above)
Comment (mivanov1988) [reply projects/control-service/projects/helm_charts/pipelines-control-service/values.yaml:810]: What do you mean?

Comment (mivanov1988) [reply projects/control-service/projects/helm_charts/pipelines-control-service/values.yaml:810]: Good catch :)

**Review by tozka — APPROVED** [review]

File: projects/control-service/projects/helm_charts/pipelines-control-service/values.yaml [hunk 3]
```diff
@@ -792,21 +792,36 @@ fluentd:
...
+    ##                            job executions exceed this number, the older executions are deleted.
+    ##
+    ## executionsTtlSeconds - Total time to live per data job execution. Data job executions older than ttlSeconds
+    ##              specified also get deleted.
+    ##
+    cleanupTask:
+      ## maximum number of data job executions to store in the database / default is 100.
+      maximumExecutionsToStore: "100"
+      ## maximum time to live seconds for each execution / default is 14 days.
+      executionsTtlSeconds: "1209600"
```
Comment (tozka) [inline projects/control-service/projects/helm_charts/pipelines-control-service/values.yaml:814]: Those are not backward compatible. I assume those have the same defaults in the Control Service if not set so the change would not cause issues (crashes) in existing deployments

Comment (mivanov1988) [reply projects/control-service/projects/helm_charts/pipelines-control-service/values.yaml:814]: Yes they have the same defaults in the Control Service - https://github.com/vmware/versatile-data-kit/blob/main/projects/control-service/projects/pipelines_control_service/src/main/resources/application.properties

**Pull request merged**